packages/components/react/src/CodeMirrorEditor/index.tsx

```diff
@@ -46,7 +46,7 @@ export type OnChangeCallback = (update: EditorUpdate) => void;
 export type OnScrollCallback = (position: ScrollPosition) => void;
 
 interface Props {
-  reset?: unknown;
+  id?: unknown;
```

Comment:
Renamed to be consistent with how `'react-resizable-panels'` uses their `id` (given that here it has the same purpose).